controller/Readme.md

```diff
@@ -28,7 +28,10 @@ Notes:
 - The controller application requires certain artifacts to be in a specific location relative to binary location. The `dune build @install` command ensures this.
 - The `--profile release` option disables dune from failing on warnings (which are currently present in the `obus` library)
 
+## Code style
+
+- Author CSS according to these [BEM-based guidelines](https://github.com/dividat/diviapps/blob/develop/frontend/docs/dev/styles.md).
```

Comment:
I think linking to private resources in a public repo isn't very nice.

Maybe just saying 

```
- Author CSS according to the [BEM methodology](http://getbem.com/) in the format `d-Block__Element--Modifier.
```
will be enough in this context. Enough information for internal devs and if we have external contributors we will have to find another method of knowledge transfer anyway.

Reply:
Right!